deployment/migrations/versions/0017_f9fa39b6bdef_vm_instances.py

```diff
@@ -129,7 +129,7 @@ def recreate_cost_views():
                                            free_disk.included_disk_space,
                                            0) AS additional_disk_space) additional_disk,
                  LATERAL ( SELECT CASE
-                                      WHEN vms.persistent THEN 2000
+                                      WHEN COALESCE(vms.persistent, true) THEN 2000
                                       ELSE 200
                                       END AS base_compute_unit_price) bcp,
                  LATERAL ( SELECT 1 + vms.environment_internet::integer AS compute_unit_price_multiplier) m,
```

Comment:
The changes to this migration script will have to be put in a separate migration script that recreates the VM cost view (views must be recreated entirely, you cannot just add a column). Create a new empty migration script with `alembic revision -m "fix VM cost view"` and then take the pieces that you need from this migration script to drop the vm cost view and recreate it.